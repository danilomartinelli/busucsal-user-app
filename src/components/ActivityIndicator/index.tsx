import React from "react";
import {
  View,
  ActivityIndicatorProps,
  ActivityIndicator as ActivityIndicatorRN,
} from "react-native";
import styles from "../../resources/styles";

interface IProps extends ActivityIndicatorProps {}

const ActivityIndicator = (props: IProps) => (
  <View style={styles.FlexCenter}>
    <ActivityIndicatorRN {...props} />
  </View>
);

export default ActivityIndicator;
