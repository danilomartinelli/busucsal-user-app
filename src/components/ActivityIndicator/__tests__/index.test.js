import React from "react";
import { shallow } from "enzyme";
import ActivityIndicator from "..";

describe("<ActivityIndicator />", () => {
  const getProps = () => ({
    color: "blue",
  });

  describe("render", () => {
    it("should render properly", () => {
      const propsMock = getProps();

      const wrapper = shallow(<ActivityIndicator {...propsMock} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
