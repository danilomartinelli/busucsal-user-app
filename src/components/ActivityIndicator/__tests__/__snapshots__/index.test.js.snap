// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActivityIndicator /> render should render properly 1`] = `ShallowWrapper {}`;
